### ISSUE TITLE
Reading: Retrofitting Word Vectors to Semantic Lexicons

### ISSUE THREAD

**Issue by a1da4** [a1da4/paper-survey#210]
## 0. Paper
@inproceedings{faruqui-etal-2015-retrofitting,
    title = "Retrofitting Word Vectors to Semantic Lexicons",
    author = "Faruqui, Manaal  and
      Dodge, Jesse  and
      Jauhar, Sujay Kumar  and
      Dyer, Chris  and
      Hovy, Eduard  and
      Smith, Noah A.",
    booktitle = "Proceedings of the 2015 Conference of the North {A}merican Chapter of the Association for Computational Linguistics: Human Language Technologies",
    month = may # "{--}" # jun,
    year = "2015",
    address = "Denver, Colorado",
    publisher = "Association for Computational Linguistics",
    url = "https://aclanthology.org/N15-1184",
    doi = "10.3115/v1/N15-1184",
    pages = "1606--1615",
}
## 1. What is it?
They proposed a new approach, post-processing, to bring words closer together

## 2. What is amazing compared to previous works?
Recent methods train the model from scratch by adding task-specific functions (ad-hoc).
In this paper, the post-processing approach can be used on any pre-trained model (post-hoc).

## 3. Where is the key to technologies and techniques?
![スクリーンショット 2021-09-29 2 28 34](https://user-images.githubusercontent.com/45454055/135136304-a771b34d-69e9-406b-91b0-ef266c87c4fb.png)

To obtain linked-vectors (white), they post-process the pre-trained word vectors (gray)

The objective function is below:
 - left: preserve pre-trained vectors 
 - right: bring the target word closer to the word taken from the external knowledge graph (e.g. WordNet)
![スクリーンショット 2021-09-29 2 30 15](https://user-images.githubusercontent.com/45454055/135136521-760d7c77-7eb5-4f6f-b42e-5abee75a3883.png)

The model is fine-tuned via online training (update per word).
![スクリーンショット 2021-09-29 2 34 08](https://user-images.githubusercontent.com/45454055/135137114-95eced72-19a5-4a4b-a48a-5079977e8ad3.png)

## 4. How did evaluate it?
Word Similarity tasks:
- their methods improve the TOEFL task (synonym detection)
- their methods do not work on the SYN-REL task (syntax)
- external knowledge graph: paraphrase database > WordNet >> FrameNet (group words based on very abstract concepts?)
![スクリーンショット 2021-09-29 2 34 51](https://user-images.githubusercontent.com/45454055/135137225-a5ee70d9-eeeb-4243-8c2e-ed6e82ea3382.png)

## 5. Is there a discussion?
Compare with other training methods:
 - Lazy: full-batch + normalization
 - Periodic: mini-batch
 - Retrofitting: online (one-batch)
 - Retrofitting achieves comparable results to Periodic.
![スクリーンショット 2021-09-29 2 39 19](https://user-images.githubusercontent.com/45454055/135137875-01b9d825-9ba6-42e9-bdbf-eb8627197311.png)

**Comment by a1da4** [comment]
#211 
+antonymy

**Comment by a1da4** [comment]
#212 
+hypernymy
